test/end-to-end/stock/stock.lots.spec.js

```diff
@@ -82,7 +82,7 @@ function StockLotsRegistryTests() {
   it('find lots by expiration date', async () => {
     await modal.setdateInterval('01/01/2017', '31/12/2017', 'expiration-date');
     await modal.submit();
-    await GU.expectRowCount(gridId, 1 + depotGroupingRow);
+    await GU.expectRowCount(gridId, depotGroupingRow - 1); // 1 expired in this date range
```

Comment:
I'm quite surprised this is the only test that needs to change.  But this is great!

Reply:
Actually I ended up changing 3-4 tests.  Not sure why this is the only one that showed up.